### PR TITLE
Update documentation related to the latest AWS Lambda runtime nodejs20.x

### LINKED ISSUE #5524: Update documentation related to the latest AWS Lambda runtime `nodejs20.x`
- When using `nodejs18.x` or down, the documentation is currently accurate.
- When using `nodejs20.x` the documentation is outdated, because the `rhel-openssl-3.0.x` `binaryTarget` should be used, if not it would error at runtime with an error like "Prisma Client could not locate the Query Engine for runtime "rhel-openssl-3.0.x"." ([Example](https://github.com/prisma/prisma/issues/22244) for Netlify)

Pages that need to be updated
- [ ] Deployment pages
	- [ ] https://www.prisma.io/docs/guides/deployment/caveats-when-deploying-to-aws-platforms#aws-lambda-upload-limit
	- [ ] https://www.prisma.io/docs/guides/deployment/serverless/deploy-to-aws-lambda
	- [ ] https://www.prisma.io/docs/guides/deployment/serverless/deploy-to-netlify
- [ ] Env var reference about the Prisma CLI
	- [ ] https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_cli_binary_targets

Sidenote:
We will add an ecosystem test in this PR https://github.com/prisma/ecosystem-tests/pull/4345

### PR DESCRIPTION
**Pull request by jharrell**
Fixes #5524

### CODE REVIEW

File: content/200-orm/500-reference/300-environment-variables-reference.mdx [hunk 1]
```diff
@@ -270,10 +270,12 @@ The `PRISMA_FMT_BINARY` variable is used in versions [4.2.0](https://github.com/
 
 `PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not affect runtime of Prisma CLI or Prisma Client).
 
-Use `PRISMA_CLI_BINARY_TARGETS` if you 1) deploy to a specific platform via an upload of a local project that includes dependencies, and 2) your local environment is different from the target (e.g. AWS Lambda is `rhel-openssl-1.0.x`, and your local environment might be macOS arm64 `darwin-arm64`). Using the `PRISMA_CLI_BINARY_TARGETS` environment variable ensures that the target engine files are also downloaded.
+Use `PRISMA_CLI_BINARY_TARGETS` if you 1) deploy to a specific platform via an upload of a local project that includes dependencies, and 2) your local environment is different from the target (e.g. AWS Lambda with Node.js 18 is `rhel-openssl-1.0.x`, and your local environment might be macOS arm64 `darwin-arm64`). Using the `PRISMA_CLI_BINARY_TARGETS` environment variable ensures that the target engine files are also downloaded.
```
Comment (mhessdev) [inline content/200-orm/500-reference/300-environment-variables-reference.mdx]: this isn't a blocker but I find this paragraph odd. Not a fan of the 1) and 2) being displayed this way in a paragraph. But I see that's how it was before, so it's not a big deal.

**Review by mhessdev — APPROVED** [review]

**Review by janpio — APPROVED** [review]
Looks fine.

Do we maybe want to document it in a way already that we do not have to updated again when Node 20 becomes the default, or more common, so to present the options more equal and maybe the more modern one first?

**Comment by jharrell** [comment]
Thanks @janpio , I made some changes making tabs where relevant and Node 20 the default where not. Hopefully that matches your expectation?

**Review by janpio — APPROVED** [review]
Looks good 👍

**Comment by janpio** [comment]
Actually: Does this also apply to the ARM Lambda? There it is still 1.0.x exclusively.
image

File: content/200-orm/500-reference/300-environment-variables-reference.mdx [hunk 2]
```diff
@@ -270,10 +270,12 @@ The `PRISMA_FMT_BINARY` variable is used in versions [4.2.0](https://github.com/
 
 `PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not affect runtime of Prisma CLI or Prisma Client).
 
-Use `PRISMA_CLI_BINARY_TARGETS` if you 1) deploy to a specific platform via an upload of a local project that includes dependencies, and 2) your local environment is different from the target (e.g. AWS Lambda is `rhel-openssl-1.0.x`, and your local environment might be macOS arm64 `darwin-arm64`). Using the `PRISMA_CLI_BINARY_TARGETS` environment variable ensures that the target engine files are also downloaded.
+Use `PRISMA_CLI_BINARY_TARGETS` if you 1) deploy to a specific platform via an upload of a local project that includes dependencies, and 2) your local environment is different from the target (e.g. AWS Lambda with Node.js is `rhel-openssl-3.0.x`, and your local environment might be macOS arm64 `darwin-arm64`). Using the `PRISMA_CLI_BINARY_TARGETS` environment variable ensures that the target engine files are also downloaded.
```
Comment (janpio) [inline content/200-orm/500-reference/300-environment-variables-reference.mdx]:
```suggestion
Use `PRISMA_CLI_BINARY_TARGETS` if you 1) deploy to a specific platform via an upload of a local project that includes dependencies, and 2) your local environment is different from the target (e.g. AWS Lambda with Node.js 20+ is `rhel-openssl-3.0.x`, and your local environment might be macOS arm64 `darwin-arm64`). Using the `PRISMA_CLI_BINARY_TARGETS` environment variable ensures that the target engine files are also downloaded.
```

**Comment by janpio** [comment]
> Actually: Does this also apply to the ARM Lambda? There it is still 1.0.x exclusively. image

What is the answer to this question?

**Comment by jharrell** [comment]
@janpio I don't know if this applies to ARM64 architecture. I don't think it does, but I will need to try it out.

**Comment by janpio** [comment]
Please do, otherwise the information in our docs is now incomplete.

**Pull request merged**